atom/browser/api/atom_api_web_contents.h

```diff
@@ -37,8 +37,9 @@ class WebContents : public mate::TrackableObject<WebContents>,
                     public content::WebContentsObserver {
  public:
   // For node.js callback function type: function(error, buffer)
-  typedef base::Callback<void(v8::Local<v8::Value>, v8::Local<v8::Value>)>
-      PrintToPDFCallback;
+  using PrintToPDFCallback =
+      base::Callback<void(v8::Local<v8::Value>, v8::Local<v8::Value>)>;
+  using JsCompletionCallback = base::Callback<void(v8::Local<v8::Value>)>;
```

Comment:
`JsCompletion` seems a more generous name since we only use it in `workspace` API, it woud be better to rename it to a more suitable name, like `WorkSpaceCallback`.